schemas/device/electrodeArrayUsage.schema.tpl.json

```diff
@@ -1,78 +1,49 @@
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/core/File",
-        "https://openminds.ebrains.eu/core/FileBundle"
-      ]
-    },
     "anatomicalLocation": {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add all anatomical entities in which this electrode array is located.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-        "https://openminds.ebrains.eu/sands/CustomAnatomicalEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntityVersion"
+      "_instruction": "Add all anatomical entities that describe the semantic location of the contact point(s) of the electrode array best.",
+      "_linkedCategories": [
+        "anatomicalLocation"
       ]
     },
     "contactResistance": {
       "type": "array",
       "minItems": 2,
       "uniqueItems": true,
-      "_instruction": "Enter the contact resistance of each electrode of this array, in the same order that has been specified by letrero identifier in electrode array card.",
+      "_instruction": "Enter the contact resistance for each electrode of this array during its use, in the same order that the electrode identifiers for this electrode array have been specified.",
       "_embeddedTypes": [
         "https://openminds.ebrains.eu/core/QuantitativeValue",
         "https://openminds.ebrains.eu/core/QuantitativeValueRange"
       ]
-    },
-    "coordinatePoint": {
+    },    
+    "device": {
+      "_instruction": "Add the electrode array used.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/ephys/ElectrodeArray"
+      ]
+    },   
+    "spatialLocation": {
       "type": "array",
       "minItems": 2,
       "uniqueItems": true,
-      "_instruction": "Add the central coordinate of each electrode of this array, in the same order that has been specified by electrode identifier in electrode array card.",
+      "_instruction": "Add all coordinate points that best describe the spatial location of each electrode contact of this array during its use, in the same order that the electrode identifiers for this electrode array have been specified.",
       "_embeddedTypes": [
         "https://openminds.ebrains.eu/sands/CoordinatePoint"
       ]
     },
-    "electrodeArray": {
-      "_instruction": "Add the electrode array used.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/ephys/ElectrodeArray"
-      ]
-    },
-    "lookupLabel": {
-      "type": "string",
-      "_instruction": "Enter a lookup label for this electrode array usage."
-    },
     "usedElectrode": {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Enter the identifier of all used electrodes.",
+      "_instruction": "Enter the identifiers of all electrodes during the use of this electrode array.",
```

Comment:
"Enter the identifiers for all electrodes used with this electrode array usage."
or maybe
"Enter the identifiers of all electrodes used during this electrode array usage."